### PR TITLE
Southern Slavic Culture Group - Merge Ready

### PR DESCRIPTION
**Pull request by Charanraja123**
Still Need to add localizations for these.

### CODE REVIEW

File: common/flavorization/00_CTP.txt [hunk 1]
```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
 ## End of Dravidian_cultrue_group ##
 ###############################################
 
+###############################################
+## South_slavic_cultrue_group ##
```
Comment (Viridianus) [inline common/flavorization/00_CTP.txt:7087]: Again, culture not cultrue should be here.

File: common/flavorization/00_CTP.txt [hunk 2]
```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
...
+	type = title
+	tier = empire
+	priority = 103
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+###############################################
+## End of South_slavic_cultrue_group ##
```
Comment (Viridianus) [inline common/flavorization/00_CTP.txt:8854]: Same here, cultrue -> culture.

File: common/flavorization/00_CTP.txt [hunk 3]
```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
...
+	tier = empire
+	priority = 103
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+emperor_feudalclan_female_bulgarian = {
+	type = character
+	gender = male
```
Comment (Viridianus) [inline common/flavorization/00_CTP.txt:8835]: This line should say gender = female.

File: common/flavorization/00_CTP.txt [hunk 4]
```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
...
+	tier = kingdom
+	priority = 48
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+king_feudalclan_female_bulgarian = {
+	type = character
+	gender = male
```
Comment (Viridianus) [inline common/flavorization/00_CTP.txt:8804]: This line should say "gender = female".

File: common/flavorization/00_CTP.txt [hunk 5]
```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
...
+	priority = 29
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+
+duke_feudalclan_female_bulgarian = {
+	type = character
+	gender = male
```
Comment (Viridianus) [inline common/flavorization/00_CTP.txt:8729]: This line should say "gender = female".

File: common/flavorization/00_CTP.txt [hunk 6]
```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
...
+	priority = 19
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+
+count_feudalclan_female_bulgarian = {
+	type = character
+	gender = male
```
Comment (Viridianus) [inline common/flavorization/00_CTP.txt:8613]: This line should say "gender = female".

**Comment by Viridianus** [comment]
I assume there may be more gender-related mistakes. Please re-read it carefully.

**Comment by Charanraja123** [comment]
> 
> 
> I assume there may be more gender-related mistakes. Please re-read it carefully.

Yeah sure. I'm still working on it, but thanks. 
Also maybe i should stop copying 'cultrues' 👎

File: common/flavorization/00_CTP.txt (hunk 1 above)
Comment (Charanraja123) [reply common/flavorization/00_CTP.txt]: Fixed

File: common/flavorization/00_CTP.txt (hunk 2 above)
Comment (Charanraja123) [reply common/flavorization/00_CTP.txt]: Fixed

File: common/flavorization/00_CTP.txt (hunk 3 above)
Comment (Charanraja123) [reply common/flavorization/00_CTP.txt]: Fixed all gender related errors. I'll resolve all of them

**Comment by Charanraja123** [comment]
I've uploaded the localisation files for the slavic culture. Please check them to see if I have missed or incorrectly entered any titles.
The blank spaces in the localisation are for titles that dont exist in the sheets. I've left them blank for review purposes. If you guys have suggestions for filling any of the blanks let me know and I'll add them in. 

** I'll remove the empty localisations and their respective defines in the cpt file once the review is done**

Thanks

**Review by Viridianus — COMMENTED** [review]
Everything looks solid but do we really want to keep those lines ending in ""?

**Comment by Charanraja123** [comment]
Definitely not. That would make all lot of titles blank since they still get called but have no localization. 
As i stated above I intend on removing them once someone reviewed and was okay with it. Which I'll start doing now

**Comment by Charanraja123** [comment]
I'm Removing duplicates, Do Not Merge

**Comment by Charanraja123** [comment]
I'm Done. I think this can be merged

**Review by Viridianus — APPROVED** [review]

**Review by Gr770 — APPROVED** [review]

**Pull request merged**